ui/v2.5/src/components/Settings/SettingsConfigurationPanel.tsx

```diff
@@ -383,9 +384,9 @@ export const SettingsConfigurationPanel: React.FC = () => {
             Directory location of the cache
           </Form.Text>
         </Form.Group>
-
+        <h4>Library File Types</h4>
         <Form.Group id="video-extensions">
-          <h6>Video Extensions</h6>
+          <h6>Included Video Extensions</h6>
```

Comment:
I don't think `Included` adds much utility personally. `Video Extensions` is good enough imo, `Video File Extensions` if you want to be really clear.